### ISSUE TITLE
Correction to execute.yaml & input.yaml defining inputs

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/ogcapi-processes#152]
The following changes correct the execute request OpenAPI schema so as not to allow an infinite nesting of input dictionaries.
The key -> input or list of inputs dictionary should only be at the execute.yaml level.

[***execute.yaml:***](https://github.com/opengeospatial/ogcapi-processes/blob/master/core/openapi/schemas/execute.yaml)
```yaml
inputs:
   additionalProperties:
      oneOf:
      - $ref: "input.yaml"
      - type: array
         items:
            - $ref: "input.yaml"
```

[***input.yaml:***](https://github.com/opengeospatial/ogcapi-processes/blob/master/core/openapi/schemas/input.yaml)
```yaml
oneOf:
  - $ref: "inlineOrRefData.yaml"
  - $ref: "boundingBoxData.yaml" 
```

**Comment by pvretano** [comment]
@jerstlouis did we decide to NOT allow infinite nesting?  I don't recall that ...

**Comment by jerstlouis** [comment]
@pvretano infinite nesting does not make any sense here.

The execute request provides a list of inputs.
For input without multiplicity, you can provide one input object for it.
For input with multiplicity, you can provide a list of inputs.

The schema currently there just didn't make sense (but it would validate, because it allowed just about anything).

**Comment by pvretano** [comment]
@jerstlouis yes, but did the rest of the SWG agree to remove this?  I am happy to remove it but not if this was not a SWG decision or a review comment that we agreed to implement.

**Comment by jerstlouis** [comment]
@pvretano I believe this is the correct schema describing what was already agreed to in the SWG.
The SWG never agreed to add this meaningless dictionary of dictionary of dictionary of inputs :)
I have also mentioned this to @bpross-52n who doesn't see a problem with the change.
I don't see this as a contentious change, but we could review the change in the SWG on Monday if you feel we should? Maybe best in this case to be a separate PR and/or commit? Thanks!

**Comment by pvretano** [comment]
@jerstlouis cool.  Let me make the change (as a separate PR) since this change might impact issue #122 and I was to try and avoid merge conflicts.

**Comment by pvretano** [comment]
Created PR https://github.com/opengeospatial/ogcapi-processes/pull/154 to resolve this issue.

**Comment by jerstlouis** [comment]
Thanks @pvretano 

Suggestion: it might be good to move the `additionalProperties:` from _output.yaml_ to _execute.yaml_, so that _output.yaml_ (spelled singular) really represents a single output, like _input.yaml_ now also represents a single output. I think that would help avoid confusion and facilitate referring to the schemas.

**Comment by pvretano** [comment]
@jerstlouis you must be reading my mind!  I was just getting ready to make the exact change!

**Comment by bpross-52n** [comment]
Should be fixed by #154
